src/main/resources/site/processors/add-metadata.ts

```diff
@@ -0,0 +1,91 @@
+import type {Request, Response} from '/lib/types';
+
+
+import {forceArray} from '@enonic/js-utils/array/forceArray';
+import {getFixedHtmlAttrsAsString} from '/lib/metadata/getFixedHtmlAttrsAsString';
+import {getMetaData} from '/lib/metadata/getMetaData'
+import {getReusableData} from '/lib/metadata/getReusableData';
+import {getTitleHtml} from '../../lib/metadata/getTitleHtml';
```

Comment:
Should probably not use the relative path? Should be the same as for the previous two imports.

Reply:
fixed